x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_delete_rules/route.ts

```diff
@@ -112,6 +100,21 @@ export const bulkDeleteRulesRoute = (router: SecuritySolutionPluginRouter, logge
     }
   };
 
-  router.delete(config, handler);
-  router.post(config, handler);
+  const routeConfig: VersionedRouteConfig<'post' | 'delete'> = {
+    access: 'public',
+    path: DETECTION_ENGINE_RULES_BULK_DELETE,
+    options: {
+      tags: ['access:securitySolution'],
+    },
+  };
+  const versionConfig = {
+    version: '2023-10-31',
+    validate: {
+      request: {
+        body: buildRouteValidation(BulkDeleteRulesRequestBody),
+      },
+    },
+  };
+  router.versioned.delete(routeConfig).addVersion(versionConfig, handler);
+  router.versioned.post(routeConfig).addVersion(versionConfig, handler);
```

Comment:
Ahhh, yeah, this was the other thing I was curious about with the above comment (WRT abstracting/isolating versioned configs). Happens organically when needing to tag multiple http verbs, LGTM! 👍 🙂